drivers/hubitat-mqtt-link-driver.groovy

```diff
@@ -136,35 +138,31 @@ def publish(topic, payload) {
 }
 
 def subscribe(topic) {
-    if (notMqttConnected()) {
```

Comment:
This subscribe() method and unsubscribe() below are tied to the buttons on the Device page and exist for troubleshooting. 

Removing this check for mqtt connection will cause the un/subscribe to fail. These should be put back to ensure that connection exists before the un/sub.